server/src/routes.ts

```diff
@@ -24,8 +25,15 @@ export function initRoutes(app: Application) {
             .isIn(Object.values(KxDocumentType)).withMessage('Invalid document type'),
         body('language').optional().notEmpty().withMessage('Language is required')
             .isString().withMessage('Language must be a string'),
-        body('area_type').notEmpty().withMessage('Area type is required')
-            .isIn(Object.values(AreaType)).withMessage('Invalid area type value'),
+        body('doc_coordinates').notEmpty().withMessage('Document coordinates are required').isObject()
+            .custom((v) => {
+                return isDocCoords(v) &&  
+                    (
+                        (v.type === AreaType.ENTIRE_MUNICIPALITY) ||
+                        (v.type === AreaType.POINT && coordDistance(v.coordinates as [number, number], KIRUNA_COORDS) < 30) ||
```

Comment:
Here it seems that we are considering 30km from the center of kiruna, I think that by doing so we are also considering as invalid points that we are still part of kiruna given the large area covered by kiruna